Lombiq.Gulp.Extensions.csproj

```diff
@@ -18,17 +18,11 @@
     <ESLintConfigFileTargetPath>../../.eslintrc</ESLintConfigFileTargetPath>
   </PropertyGroup>
   
-  <Target Name="SelectPnpmVersion" BeforeTargets="InitCommonNodeModules">
-    <!-- Ignore sporadic errors when this is run by multiple threads at the same time. -->
-    <Exec Command="corepack enable"
-          IgnoreExitCode="true"
-          IgnoreStandardErrorWarningFormat="true"
-          StandardOutputImportance="$(NxVerbosity)" />
-    <!-- Enable the latest version of pnpm@6. -->
-    <Exec Command="corepack prepare pnpm@latest-7 --activate"
-          IgnoreExitCode="true"
-          IgnoreStandardErrorWarningFormat="true"
-          StandardOutputImportance="$(NxVerbosity)" />
+  <Target Name="ActivatePnpmVersion7" BeforeTargets="InitCommonNodeModules">
+    <!-- Corepack is a Node.js tool available since v14.9.0 and v16.9.0. -->
+    <Exec Command="corepack enable" />
+    <!-- Enable the latest version of pnpm@7. -->
+    <Exec Command="corepack prepare pnpm@latest-7 --activate" />
   </Target>
```

Comment:
@dministro @0liver Just for info.

In the Folkways repo we already updated GE to fix MSB3073 errors.

But with this last changes, locally under Visual Studio on `corepack enable` I got an "operation not permitted".

To get it working I needed to run Visual Studio as administrator.

But then on `corepack prepare` I got an "incorrect semver for pnpm@latest-7" (e.g. "pnpm@7.30.5" works).

So I reverted to the previous commit that works, maybe because of the additional attributes in the `<Exec />` cmds.

Reply:
We've found out that the second problem you faced, @jtkech, is due to an older Node.js version you must be using. It seems that support for the `latest-7` tag is part of Node.js 18.

We've adjusted the code to use `7.30.5` explicitly now, which should work just fine.